Sample[500] samples
